test/model/content_test.dart

```diff
@@ -344,6 +344,14 @@ class ContentExample {
       ]),
     ]);
 
+  static final codeBlockWithEmptyBody = ContentExample(
```

Comment:
Let's have a comment that makes it clear we don't expect this in messages from current servers. The link to the ancient message you put in the PR description is helpful; we could include that here too.